lib/index.js

```diff
@@ -16,15 +16,15 @@ var runQuery = function (credentials, method) {
     var p = new Promise(function(resolve, reject) {
       var success = function(results) {
         if (typeof cb === 'function') {
-          cb.apply(null, [null].concat(Array.prototype.slice.call(arguments)));
+          cb(null, results);
         }else{
```

Comment:
I like the idea, @t3chnoboy @masterT  what style guide do you recommend? 

i propose the airbnb styleguide https://github.com/airbnb/javascript